src/NServiceBus.AcceptanceTests/Reliability/Outbox/When_a_duplicate_message_arrives.cs

```diff
@@ -0,0 +1,125 @@
+﻿namespace NServiceBus.AcceptanceTests.Reliability.Outbox
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NServiceBus.AcceptanceTests.ScenarioDescriptors;
+    using NServiceBus.Configuration.AdvanceExtensibility;
+    using NServiceBus.Pipeline;
+    using NUnit.Framework;
+
+    public class When_a_duplicate_message_arrives : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task It_does_not_dispatch_messages_aready_dispatched()
```

Comment:
I am not sure yet how to improve that API. With the new "nullify the operations on dispatcihng" approach, it is super easy to just return an empty list of transport operations to the core (if they are dispatched).